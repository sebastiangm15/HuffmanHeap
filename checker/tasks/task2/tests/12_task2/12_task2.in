16
9 RDT78
15 FTA35
27 JRV99
32 BKH93
37 TXP15
39 OPM23
42 UDL48
48 POX14
50 RYH20
51 RJR26
55 TFU85
56 MTY73
67 QXZ23
77 NWE79
87 SCC16
100 WRI11
4
0001010011010001011111011111
1100001010100001101111011100010
010001000010101
10011101100
