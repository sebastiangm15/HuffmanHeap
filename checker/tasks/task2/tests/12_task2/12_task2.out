NWE79 FTA35 RDT78 RJR26 QXZ23 TXP15 
TFU85 OPM23 RDT78 POX14 BKH93 QXZ23 OPM23 
SCC16 OPM23 NWE79 JRV99 
WRI11 QXZ23 TFU85 
